Sources/Plasma/PubUtilLib/plAvatar/plAvBrainHuman.cpp

```diff
@@ -304,20 +299,14 @@ void plAvBrainHuman::IInitBoneMap()
         { RThumb3,          "Bone_RThumb3" },
     };
 
-    int numTuples = sizeof(tupleMap) / sizeof(tuple);
-
-    for(int i = 0; i < numTuples; i++)
+    for (const auto& [BoneID, BoneName] : BoneMap)
     {
-        HumanBoneID id = tupleMap[i].fID;
-        ST::string name = tupleMap[i].fName;
-        
-        const plSceneObject * bone = this->fAvMod->FindBone(name);
-        if( bone )
+	    if( const plSceneObject * bone = this->fAvMod->FindBone(BoneName) )
         {
-            fAvMod->AddBoneMapping(id, bone);
+            fAvMod->AddBoneMapping(BoneID, bone);
         }
         else
-            hsStatusMessageF("Couldn't find standard bone %s.", name.c_str());
+            hsStatusMessageF("Couldn't find standard bone %s.", BoneName.data());
```

Comment:
While we're here, let s fix this bracing to match [Plasma Code Style](https://github.com/H-uru/Plasma/wiki/Preferred-Code-Style). We prefer for all branches to use brackets if any one branch requires them.